src/main/java/com/example/nlern/repos/QuestionRepository.java

```diff
@@ -2,8 +2,16 @@
 
 
 import com.example.nlern.models.Question;
+import jakarta.transaction.Transactional;
 import org.springframework.data.repository.CrudRepository;
 
+import java.util.List;
+
 public interface QuestionRepository extends CrudRepository<Question, Integer> {
+    List<Question> findByTestIdOrderById(int Test);
```

Comment:
Same here
int Test, isn't Test a whole class?

Reply:
Yes, it is. Corrected